fractal_server/images/models.py

```diff
@@ -69,6 +70,11 @@ def validate_attributes(
                 )
         return v
 
+    @validator("zarr_url", "origin")
```

Comment:
move to base

split into two, for the two attributes, so that we have strict types